### PR TITLE
Clean up CSS and category layout

### PR DESCRIPTION
**Pull request by sarinac**
**What Changed:**
99% of edits was cleaning up the CSS code. No action needed on your end but just documenting for visibility.

Details... 
* Edited `main.css` by only keeping what's important (and removing 10k unnecessary lines...). It's now easy to manage!
* Deleted `jupyternotebook.css` because it contained too much code and was probably unnecessary. I moved the main chunks into `main.css`.
* Rename `sidebar.html` into `categories.html` since it's the category filter for your blog posts.
* Made footer much simpler in layout and renamed classes (`footer.html`)

**What's next?**
We can customize your About page now that the CSS cleanup is out of the way! :D

### CODE REVIEW

**Comment by canyon289** [comment]
Sorry for delay

File: dev/custom/templates/fragments/categories.html [hunk 1]
```diff
@@ -0,0 +1,10 @@
+<div class="categories-menu">
```
Comment (canyon289) [inline dev/custom/templates/fragments/categories.html:1]: What does this fragment do?

Comment (canyon289) [reply dev/custom/templates/fragments/categories.html:1]: Ah this is rename nevermind

**Comment by canyon289** [comment]
Thank you! PRs that look like this are the best
image

File: dev/custom/templates/base.html [hunk 2]
```diff
@@ -47,29 +46,18 @@
 
 		{% include 'fragments/main-menu-mobile.html' %}
 
-		<div class="wrapper">
+		<div class="wrapper body-wrapper">
```
[the remark this answers is not in the document]
Comment (sarinac) [reply dev/custom/templates/base.html:49]: The `body-wrapper` class will make sure that your footer is at the bottom of the window by making your page's min-height take up the entire window. If I don't have this min-height constraint, then your footer will appear in the middle of the screen for super short articles.
https://github.com/sarinac/canyon289.github.io/blob/f9c2423bd6d554abf8121b6cefdaaa3a2d5f32a3/dev/custom/static/css/main.css#L102

[the remark this answers is not in the document]
Comment (canyon289) [reply dev/custom/templates/base.html:49]: Thanks for the explanation

**Pull request merged**